features/admin.sms-templates.v1/pages/sms-customization.tsx

```diff
@@ -180,7 +186,11 @@ const SMSCustomizationPage: FunctionComponent<SMSCustomizationPageInterface> = (
 
         if (smsTemplateError.response.status === 404) {
             setIsTemplateNotAvailable(true);
-            if (!isSystemTemplate || selectedLocale !== SMSTemplateConstants.DEAFULT_LOCALE) {
+            if (isSubOrgUser && !isInheritedTemplate) {
```

Comment:
Do we really need a new state variable `isInheritedTemplate`? I think using `isSubOrgUser` is sufficient.

Reply:
Yes we will need `isInheritedTemplate`, otherwise this block will be true every time. We need to pass to next block if fetching inherited template fails.